doc/source/CHANGELOG.rst

```diff
@@ -16,6 +16,8 @@ Changelog
 - msm: fixed serialization of BayesianHMSM, if initialized with a ML-HMSM. #1283
 - base: require at least tqdm >= 4.23, because of an API change. #1292, #1293
 - plots: fixed minor bug in plot_network (state_labels=None would not work). #1306
+- plots: refactored plots2d
```

Comment:
this message is kind of uninformative to the user. I would state, that the gca behaviour has changed etc.